metron-platform/metron-solr/README.md

```diff
@@ -0,0 +1,52 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Solr in Metron
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Installing](#installing)
+
+## Introduction
+
+Metron ships with Solr 6.6.2. Solr Cloud can be used as the real-time portion of the datastore resulting from [metron-indexing](../metron-indexing/README.md).
```

Comment:
Can you make this "... Solr 6.6.2 support", so it's obvious we're not actually including Solr?

I'd also add the blurb / link from the script about production use here as well.